src/extension.ts

```diff
@@ -1,23 +1,22 @@
 import * as vscode from 'vscode';
 import * as path from 'path';
-import { walk } from './utils';
+import { walk, getSnypetConfig, getVscodeCurrentPath } from './utils';
 
 import { parseComponents } from './component-parser';
 
-export function activate(context: vscode.ExtensionContext) {
-  // Get files having components
-  // TODO: enhance the logic to get components
-  const packageDir = 'packages';
-  const rootPath = vscode.workspace.rootPath;
```

Comment:
roothPath is deprecated https://code.visualstudio.com/api/references/vscode-api#workspace